### PR TITLE
Docs: Restructure ioping page

### PR DESCRIPTION
**Pull request by kickoke**
##### Summary
The branch has a the name "test-collector-template" because that is what I intended to do. I wanted to check out if the JS component for the collector pages would also make sense for the plugin pages. 

Well, they don't work there, but in the mean time I was so busy with improving the ioping page, that I thought I'll push my changes anyways. 😁 

##### Test Plan

When reviewing, could you check: 
- Line 28: Do I need to restart Netdata so the change comes into effect? 
- Table with metrics: I wasn't able to find any ioping charts on our servers. Could you check if the information in the table is correct? 

##### Additional Information
N/A

<details> <summary>For users: How does this change affect me?</summary>
 Improved user experience when reading the ioping documentation. 
</details>

### CODE REVIEW

File: collectors/ioping.plugin/README.md [hunk 1]
```diff
@@ -1,86 +1,120 @@
...
-A recent version of `ioping` is required (one that supports option `-N`).
-The supplied plugin can install it, by running:
+The collector is enabled if Netdata detects that you have `ioping` installed. Otherwise, it is disabled. 
 
-```sh
-/usr/libexec/netdata/plugins.d/ioping.plugin install
-```
+## Prerequisites
+
+To use this plugin, you need to install `ioping`.
```
Comment (ilyam8) [inline collectors/ioping.plugin/README.md:16]:
```suggestion
To use this plugin, you need to install [ioping](https://github.com/koct9i/ioping#ioping).
```

Took libery to make awesome more awesome 😄

Comment (kickoke) [reply collectors/ioping.plugin/README.md:16]: Thanks, that's awesome!

File: collectors/ioping.plugin/README.md [hunk 2]
```diff
@@ -1,86 +1,120 @@
...
+
+   ```bash
+   sudo ./edit-config ioping.conf
+   ```
+ ### Example
 
 ```sh
-# uncomment the following line - it should already be there
+# Uncomment the following line
 ioping="/usr/libexec/netdata/plugins.d/ioping"
```
Comment (ilyam8) [inline collectors/ioping.plugin/README.md:53]: Let's remove the line. The [default value](https://github.com/netdata/netdata/blob/31c4623cdf062b29675b612391d77b2f7f572a41/collectors/ioping.plugin/ioping.plugin.in#L163) is correct regardless of the install prefix.

File: collectors/ioping.plugin/README.md [hunk 3]
```diff
@@ -1,86 +1,120 @@
...
+
+You can install `ioping` using the plugin.
+The following command will download, build and install the correct `ioping` version as `/usr/libexec/netdata/plugins.d/ioping`.
+
+To install `ioping`, run:
+
+   ```sh
+   /usr/libexec/netdata/plugins.d/ioping.plugin install
+   ```
+   Once `ioping` is installed, Netdata will automatically start collecting disk latency data.
```
Comment (ilyam8) [inline collectors/ioping.plugin/README.md:28]: After installing we need:

- configure "destination".
- restart netdata service.

File: collectors/ioping.plugin/README.md [hunk 4]
```diff
@@ -1,86 +1,120 @@
...
+   Netdata will detect the new plugin and start it.
 
-That's it. Netdata will detect the new plugin and start it.
+## Metrics and Alerts produced by this collector
 
-You can name the new plugin any name you like.
-Just make sure the plugin and the configuration file have the same name.
+| Chart  | Metrics        | Alert                                                                             |
+| ------ | -------------- | --------------------------------------------------------------------------------- |
+| ioping | ioping.latency | [ioping_disk_latency](https://community.netdata.cloud/t/ioping-disk-latency/2120) |
```
Comment (ilyam8) [inline collectors/ioping.plugin/README.md:114]:
```suggestion
| read_latency | ioping.latency | [ioping_disk_latency](https://community.netdata.cloud/t/ioping-disk-latency/2120) |
```

I am not sure what "chart" is, but "read_latency" is closer to the truth.

Comment (ilyam8) [reply collectors/ioping.plugin/README.md:114]: > [ioping_disk_latency](https://community.netdata.cloud/t/ioping-disk-latency/2120)

I think we need 2 links:
 - [conf](https://github.com/netdata/netdata/blob/master/health/health.d/ioping.conf)
 - [troubleshoot](https://community.netdata.cloud/t/ioping-disk-latency/2120)

File: collectors/ioping.plugin/README.md (hunk 3 above)
Comment (kickoke) [inline collectors/ioping.plugin/README.md:28]:
```suggestion
   Once `ioping` is installed, you need to [restart Netdata](https://learn.netdata.cloud/docs/configure/start-stop-restart) so the new collector is recognized. 
```

Comment (kickoke) [reply collectors/ioping.plugin/README.md:28]: Configuration is covered in the next sections, but let @DShreve2 know if there is info missing.

File: collectors/ioping.plugin/README.md (hunk 2 above)
Comment (kickoke) [inline collectors/ioping.plugin/README.md:53]:
```suggestion
```

Comment (kickoke) [reply collectors/ioping.plugin/README.md:53]: Thanks for the info.

**Comment by kickoke** [comment]
@DShreve2 Can you take over? :)

File: collectors/ioping.plugin/README.md (hunk 3 above)
Comment (ilyam8) [reply collectors/ioping.plugin/README.md]: I answered to

> Line 28: Do I need to restart Netdata so the change comes into effect?

**Comment by ilyam8** [comment]
Closing because the PR is not relevant anymore. cc @tkatsoulas

**Pull request closed without merging**